python/cernbox_utils/ns_inspect.py

```diff
@@ -45,22 +44,21 @@ def _get_eos_machine(self, path):
 
     def _get_command(self, path):
         eos_machine = self._get_eos_machine(path)
-        return "/usr/bin/eos-ns-inspect dump --path %s --members eos%s-qdb:7777 --password-file /keytabs/%s_keytab --no-files --attr-query sys.acl" % (path, eos_machine, eos_machine)
+        return "/usr/bin/eos-ns-inspect scan --path %s --members eos%s-qdb:7777 --password-file /keytabs/%s_keytab --no-files --json" % (path, eos_machine, eos_machine)
 
     def _run_script(self, path):
         eos_command = self._get_command(path)
         # machine_command = "/usr/bin/ssh -oBatchMode=yes -oConnectTimeout=5 -oStrictHostKeyChecking=no -q -l root %s %s" % (self.config['nsinspect-machine'], eos_command)
-        return cernbox_utils.script.runcmd(eos_command.split(" "),shell=False)
+        return cernbox_utils.script.runcmd(eos_command.split(" "), echo=False, shell=False)
 
     def _parse_output(self, output):
         to_return = []
-        for line in output.splitlines():
-            line = line.strip()
-            search = re.search(NSOutput, line)
-            if search:
-                acl = search.group(1)
-                path = search.group(2)
-                to_return.append((path, acl))
+
+        folders = json.loads(output)
+        for folder in folders:
+            if 'xattr.sys.acl' not in folder:
```

Comment:
Can we log these ones? By design this should not happen so would be nice to keep track of those and to fix them also.

Reply:
These are essentially .sys# files. I will put the logger there

Reply:
@labkode Added the logging.